subworkflows/nf-core/utils_references/meta.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "utils_references"
+description: Functionality for dealing with references that may be useful for any Nextflow pipeline
+keywords:
+  - utility
+  - pipeline
+  - references
+components: []
+input: []
```

Comment:
You need to sort this out, yes?

Reply:
Actually no.
I just added a workflow because I couldn't properly tests the functions.
The actual workflow is never used, I'm just using the functions and the assets.